server/channels/app/slashcommands/command_test.go

```diff
@@ -416,7 +422,8 @@ func TestDoCommandRequest(t *testing.T) {
 		server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Add("Content-Type", "application/json")
 
-			io.Copy(w, InfiniteReader{})
+			_, err := io.Copy(w, InfiniteReader{})
```

Comment:
I think it's fine to just ignore the error. The discussion takes too much energy.